app/react/containers/message_form.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import axios from 'axios';
+import { connect } from 'react-redux';
+
+class MessageForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { message: '' };
+    this.onFormSubmit = this.onFormSubmit.bind(this);
+    this.onInputChange = this.onInputChange.bind(this);
+  };
+
+  onFormSubmit(event) {
+    event.preventDefault();
+    axios.post('http://localhost:3000/messages', {
+      content: this.state.message,
+      room_id: this.props.activeRoomId,
+    });
```

Comment:
I think that's completely fine. Actions are objects describing side effects launched from UI. These side effects mostly change state, but it's not required I think.